### ISSUE TITLE
Small typographic suggestions

### ISSUE THREAD

**Issue by brianloveswords** [robconery/feedback#154]
First thank you so much for writing the book I've always wanted to read. I always been fascinated with computers and programming but never formally studied them (I decided on creative writing and philosophy instead), yet somehow for the last 12 years I've been typing strange symbols into rectangles for money.

This issue is to point out some small typographic suggestions for your consideration:
- Use [em dashes](http://thewritepractice.com/what-the-heck-is-an-em-dash/) for non-parenthetical asides instead of hyphens.
- Use [curly quotes](http://practicaltypography.com/straight-and-curly-quotes.html) instead of straight quotes.
- Use [real ellipses](http://www.fileformat.info/info/unicode/char/2026/index.htm) (…) instead of three full-stops.

Thanks again for taking the time to write this book.

**Comment by robconery** [comment]
Thanks - this is one of the primary problems I have right now: everything I use for publishing blows :). I've used Scrivener which solves a lot of what you're suggesting here, but it sucks at formatting code. I used InDesign and iBooks Author which have the same problem as Scrivener.

I then used Gitbook which is OK, but doesn't do the things you're suggesting here. I've used Middleman/Jekyll... same deal.

It really isn't easy doing this stuff - but I'll try to solve it once the text gets finished up. Thanks for the nudge :).
